src/data/nat/totient.lean

```diff
@@ -351,4 +351,21 @@ begin
   apply mul_le_mul_left' (nat.totient_le d),
 end
 
+lemma totient_dvd_of_dvd (a b : ℕ) (h : a ∣ b) : φ a ∣ φ b :=
```

Comment:
There is a generic lemma about divisibility of `finsupp.prod`s behind this lemma. Please extract it and use it in the proof.

Reply:
Ah, I hadn't spotted that, thanks!

Reply:
Is this (`prod_dvd_prod_of_subset_of_dvd`) something like what you were thinking of?

Reply:
And is it ok to keep both the generic lemma and the original `totient_dvd_of_dvd` in the same PR, or would it be better to split them into two separate PRs?